package.json

```diff
@@ -45,5 +53,17 @@
     "ts-node": "^10.4.0",
     "typechain": "^8.0.0",
     "typescript": "^4.5.4"
+  },
+  "description": "A Gnosis Module framework",
+  "homepage": "https://github.com/decent-dao/fractal-contracts/blob/develop/README.md",
+  "author": "fractal-contributors",
+  "license": "MIT",
```

Comment:
Shouldn't this be "Unlicense"?

Reply:
Yeah: https://github.com/decent-dao/fractal-contracts/blob/develop/LICENSE

We should update the license in npm to that as well:
https://www.npmjs.com/package/@fractal-framework/fractal-contracts

Reply:
Re-using approved code from previous fractal-contracts-repo that Adam already approved.

Reply:
Yeah scratch that, we're going to use MIT license for this.

@christopherdancy can you also update the LICENSE file to use what we have in the frontend repo:

https://github.com/decent-dao/fractal-interface/blob/develop/LICENSE

Reply:
We should probably update the license at the top of each Solidity file to MIT in this PR then as well if we're moving to that license